lib/cropperx.dart

```diff
@@ -168,6 +168,15 @@ class _CropperState extends State<Cropper> {
                   color: widget.overlayColor,
                 ),
               ),
+            if (widget.overlayType == OverlayType.rectangle)
```

Comment:
Since the overlay frame is shared with the circle variant above, it might be nice to add it to that if-statement instead. `isCircle` will then be false since the overlay type is rectangle.